### DISCUSSION TITLE
Problem with Library

### DISCUSSION THREAD

**Discussion by Gloria0117** [mdbergmann/Eloquent#78] [Q&A]
Hi～

What could I do if I want to get the other Bible texts ? I want to get the NET/CNET/NIV/UVCB/NA28/BHS and so on. If I should get some sources of these texts first, how can I get this sources?
Any answers should be appreciated, thanks !

**Comment by mdbergmann** [comment]
Hello.

Under normal circumstances you would use the "Module installer" that is available in the app menu below "About Eloquent".

In there you'd first choose "Sync install sources" which will add a number of different sources.
Then you'd refresh each install source.

The NET bible text is available under the "eBible.org" install source.
I'm not sure about the other modules you are mentioning.
Eloquent is a just a viewer for modules made for CrossWire's SWORD library. So whatever modules are available there you will be able to use in Eloquent.
You can checkout the standard modules here: http://www.crosswire.org/sword/modules/index.jsp

**Reply by Gloria0117** [reply]
Thanks a lot.
